routes/web.php

```diff
@@ -7,7 +7,3 @@
 Route::group(['domain' => config('app.url'), 'middleware' => ['admin']], function () {
     require base_path('routes/web-admin.php');
 });
-
-Route::group(['domain' => 'vats.im'], function () {
```

Comment:
Why are we removing this?

Reply:
No longer hosted by VATSIM UK or Core, now its own service at https://vats.im/ and https://github.com/NFarrington/vatsim-url-shortener